core-models/ocrd_models/model/ocrd_page.py

```diff
@@ -24,8 +25,7 @@
     TextLineType,
     WordType,
 )
-from ocrd.constants import NAMESPACES, VERSION, MIMETYPE_PAGE
-from ocrd.model.ocrd_exif import OcrdExif
+from .ocrd_exif import OcrdExif
```

Comment:
It's a class wrapping enhancing a PIL: image (the argument to the constructor) with image metadata generated by PIL. It can be serialized and generated from a filename. Data is generated on construction.

But it would be better to split the dumb data container class and the code requiring PIL. TODO